OpenFn/Lightning
# Save adaptor icons locally

## Notes for the reviewer

- [x] Download `tar.gz.` from `https://github.com/OpenFn/adaptors/archive/refs/heads/main.tar.gz`
- [x] Copy `{rectangle,square}.png` files from `packages/*/assets/*` to `priv/static/assets/images/adapters/<adapter name>-{rectangle,square}.png`
- [x] Change logic in the app to load icons
- [ ] Maybe warn when icons haven't been downloaded?
- [x] What should we do in the case there's no internet and icons were not downloaded? If the icons haven't been downloaded, the `adaptor` name is used instead.
- [x] Add relevant tests


## Related issue

Fixes #943 

## How to Test

Initially, only `dhis2` and `common` adapters had icons. This branch downloads all the icons from the `adaptors` repo.

- If using `Docker`, make sure you rebuild the image: `docker-compose up web --build`. You can comment out `line 61`, `RUN mix lightning.install_adaptor_icons` to check that the adaptor name is used if the icons haven't been downloaded.

- When running locally, run `mix lightning.install_runtime`.


## Review checklist
